govuk-one-login/authentication-api
# [AUT-2485] Improve Dev Deployment

## What?

- Implement a python script to set up AWS CLI with well-known profiles
- Pull out common code from deploy-authdevs.sh and deploy-sandpit.sh
  into a common script
- Pull out AWS credential export into a separate script

## Why?

Local profile names were different for each person's laptop. This standardises them.

Also, it makes the deploy scripts easier to maintain.

## How to review

- code review the python script `./scripts/set-up-sso.py`
- Run the script and let it create your SSO profiles
- deploy something to sandpit - check there are no errors
- deploy something to an authdev - check there are no errors

## Related PRs

alphagov/di-infrastructure#576
govuk-one-login/authentication-frontend#1420
